src/menu/menu.module.css

```diff
@@ -0,0 +1,154 @@
+.menuList {
```

Comment:
Considerable amount of hard-coded values exist that could potential become variables to improve readability and potential customization. For example `--reactist-menulist-padding`, `--reactist-menuitem-margin`, ...

Reply:
Agree. But I'd wait until we figure out we need them. Up until now we do not anticipate the need to customize the spacing on menu elements in different contexts.